pocs/utils/rs232.py

```diff
@@ -148,13 +158,27 @@ def get_reading(self):
 
     def clear_buffer(self):
         """ Clear Response Buffer """
-        count = 0
-        while self.ser.inWaiting() > 0:
-            count += 1
-            self.ser.read(1)
-
+        # Not worrying here about bytes arriving between reading in_waiting
+        # and calling reset_input_buffer().
+        # Note that Wilfred reports that the Arduino input can seriously lag behind
+        # realtime (e.g. 10 seconds), and that clear_buffer may exist for that reason
+        # (i.e. toss out any buffered input, and then read the next full line...
+        # which likely requires tossing out a fragment of a line).
+        count = self.ser.in_waiting
+        self.ser.reset_input_buffer()
         # self.logger.debug('Cleared {} bytes from buffer'.format(count))
```

Comment:
IIRC this was generating too many log entries and so was commented out. Could consider eliminating along with `count` variable above.

Reply:
I can do so now if you like.

Reply:
Yeah, let's get rid of it.

Reply:
Done.